### ISSUE TITLE
Media encoder should expose its state in an API

### ISSUE THREAD

**Issue by toresbe** [Frikanalen/frikanalen#46]
Ideally the user should be able to monitor the progress of their files' encodes using the normal web site.

**Comment by odinho** [comment]
Indeed. This is still not in `move_and_process` which handles this encoding.
